[ESN-2880] Update Python API example to python3

javier-tello opened the pull request:
<!--- Title Link to Jira Ticket - UPDATE WITH YOUR TICKET TAG in name and url --->
## [ESN-2880](https://opengovinc.atlassian.net/browse/ESN-2880)

## Description
<!--- Describe these changes in detail --->
This PR updates the python example under the "Data API" button to python3 from python2.

## Test Procedure
<!--- List the steps involved to test the changes --->
1. Checkout the branch for this PR and run `python setup.py develop`
2. In your ini enable the `oodata` plugin.
3. Start CKAN, navigate to an existing resource or create a dataset with one one.
4. Click the "Dataset API" button when viewing the resource.
5. Scroll down to bottom and try to use the Python example in python3/


## Approval Criteria 
<!--- Describe the expected results of testing --->
- Code in example is correct and works

## Submitter Checklist
- [ ] The code looks good to me (LGTM)
- [ ] I have tested the changes locally
- [ ] The new changes does not affect web accessibility

Commit:
Update API pythoon example to python3

jguo144 commented on ckanext/odata/templates/ajax_snippets/api_info.html:
```diff
@@ -138,10 +138,10 @@ <h3>
         <div class="accordion-inner">
           <pre>
-import urllib
+from urllib import request
```
I recommend changing this to `import urllib.request`. Users may confuse this with the different `requests` module.

jguo144 commented on ckanext/odata/templates/ajax_snippets/api_info.html:
```diff
@@ -138,10 +138,10 @@ <h3>
 url = '{{ h.url_for(qualified=True, controller='api', action='action', ver=3, logic_function='datastore_search', resource_id=resource_id, limit=5) + '&q=title:jones' }}'  {# not urlencoding the ":" because its clearer #}
-fileobj = urllib.urlopen(url)
-print fileobj.read()
+fileobj = request.urlopen(url)
```
Suggestion to change this to `urllib.request.urlopen(url)` to prevent confusion with `requests` module for those new to Python.

jguo144 commented:
I think a better python example would import and use `json` for API response.

javier-tello committed:
Update sample Python Example based on feedback

changed import to urllib.request and included json library to easier parse response

jguo144 committed:
Fix indentation
